app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -0,0 +1,33 @@
+module ManagerRefresh::Inventory::AutomationManager
+  extend ActiveSupport::Concern
+  include Core
+
+  class Inventory < ManagerRefresh::Inventory::Core::Inventory
+    def credentials
```

Comment:
hm, right now I have manager level overwrite and persister level, maybe it's enough to have one. But then a manager specific options will be duplicated